### PR TITLE
replace object consisted by denom and amount to Coin

### PR DESCRIPTION
**Pull request by gnujoow**
#6

### CODE REVIEW

File: packages/core/src/apis/cosmos/bank/types.ts [hunk 1]
```diff
@@ -1,26 +1,19 @@
 import type { PaginationResponse } from "../../../types";
-
-interface Balance {
-  denom: string;
-  amount: string;
-}
+import { Coin } from "../../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/cosmos/bank/types.ts]:
```suggestion
import type { Coin } from "../../../types/common";
```

File: packages/core/src/apis/cosmos/distribution/types.ts [hunk 2]
```diff
@@ -1,31 +1,20 @@
 import { PaginationResponse } from "../../../types";
+import { Coin } from "../../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/cosmos/distribution/types.ts]:
```suggestion
import type { Coin } from "../../../types/common";
```

File: packages/core/src/apis/cosmos/governance/types.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import type {
   ProposalStatus,
   VoteOption,
 } from "../../../types";
+import { Coin } from "../../osmosis/lockup/types";
```
Comment (Ryz0nd) [inline packages/core/src/apis/cosmos/governance/types.ts:6]:
```suggestion
import type { Coin } from "../../osmosis/lockup/types";
```

File: packages/core/src/apis/cosmos/staking/types.ts [hunk 4]
```diff
@@ -1,15 +1,13 @@
 import type { BondStatus, PaginationResponse } from "../../../types";
+import { Coin } from "../../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/cosmos/staking/types.ts]:
```suggestion
import type { Coin } from "../../../types/common";
```

File: packages/core/src/apis/cosmos/tx/types.ts [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import type { BroadcastMode, SignMode } from "../../../types";
+import { Coin } from "../../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/cosmos/tx/types.ts]:
```suggestion
import type { Coin } from "../../../types/common";
```

File: packages/core/src/apis/osmosis/claim/types.ts [hunk 6]
```diff
@@ -1,3 +1,5 @@
+import { Coin } from "../../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/osmosis/claim/types.ts]:
```suggestion
import type { Coin } from "../../../types/common";
```

File: packages/core/src/apis/osmosis/gamm/types.ts [hunk 7]
```diff
@@ -1,4 +1,5 @@
 import type { PaginationResponse } from "../../../types";
+import { Coin } from "../../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/osmosis/gamm/types.ts]:
```suggestion
import type { Coin } from "../../../types/common";
```

File: packages/core/src/apis/osmosis/incentives/types.ts [hunk 8]
```diff
@@ -1,4 +1,5 @@
 import type { PaginationResponse } from "../../../types";
+import { Coin } from "../../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/osmosis/incentives/types.ts]:
```suggestion
import type { Coin } from "../../../types/common";
```

File: packages/core/src/apis/osmosis/superfluid/types.ts [hunk 9]
```diff
@@ -1,10 +1,12 @@
+import { Coin } from "../../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/osmosis/superfluid/types.ts]:
```suggestion
import type { Coin } from "../../../types/common";
```

File: packages/core/src/apis/staking/types.ts [hunk 10]
```diff
@@ -1,3 +1,5 @@
+import { Coin } from "../../types/common";
```
Comment (Ryz0nd) [inline packages/core/src/apis/staking/types.ts]:
```suggestion
import type { Coin } from "../../types/common";
```

**Review by Ryz0nd — APPROVED** [review]

**Comment by gnujoow** [comment]
굿

**Pull request merged**